### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by GiuseppeTG**
In this branch, I made some modifications to my code to improve the accessibility of my portfolio.

- Page titles - no issues found
- Image text alternatives - change them to be more descriptive
- Text headings - no issues found
- Color contrast - no issues found
- Resize - increase the size of the footer icons in the laptop version
- Interaction - add a tag for icons in the footer to make them work properly 
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,7 +67,7 @@ <h2>Projects</h2>
         
         
           <div class="project-container project-1">
-            <img src="images/project-1.jpg" alt="project-1 image">
+            <img src="images/project-1.jpg" alt="Hands typing on a macbook pro 13'', a notebook, an orange, a pen and a smartphone">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.

File: index.html [hunk 2]
```diff
@@ -87,7 +87,7 @@ <h2>Project name goes here</h2>
       <div class="container-2">
         <div class="group-2-3">
           <div class="project-container project-2">
-            <img class="project" src="images/project-2.jpg" alt="project-2 image">
+            <img class="project" src="images/project-2.jpg" alt="macbook pro 16 frontal view">
 
             <div class="single-project-info">
                   <h2>Project name goes here</h2>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.

File: index.html [hunk 3]
```diff
@@ -101,7 +101,7 @@ <h2>Project name goes here</h2>
             </div>
             
           <div class="project-container project-3">
-            <img src="images/project-3.jpg" alt="project-3 image">
+            <img src="images/project-3.jpg" alt="macbook pro 16 frontal view and a smartphone">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.

File: index.html [hunk 4]
```diff
@@ -117,7 +117,7 @@ <h2>Project name goes here</h2>
           
         <div class="group-4-5">
           <div class="project-container project-4">
-            <img src="images/project-4.jpg" alt="project-4 image">
+            <img src="images/project-4.jpg" alt="macbook pro 16">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.

File: index.html [hunk 5]
```diff
@@ -131,7 +131,7 @@ <h2>Project name goes here</h2>
           </div>
 
           <div class="project-container project-5">
-            <img src="images/project-5.jpg" alt="project-5 image">
+            <img src="images/project-5.jpg" alt="macbook pro 16 standing on its corner">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.

File: index.html [hunk 6]
```diff
@@ -151,7 +151,7 @@ <h2>Project name goes here</h2>
         <div class="container-3">
 
           <div class="project-container project-6">
-            <img class="project-image" src="images/project-6.jpg" alt="project-6 image">
+            <img class="project-image" src="images/project-6.jpg" alt="macbook pro 16 with smartphone">
 
             <div class="single-project-info">
               <h2 class="project-name">Project name goes here</h2>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @GiuseppeTG and @ElsonOtake , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @GiuseppeTG and @ElsonOtake,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**